packages/components/src/tab-panel/README.md

```diff
@@ -1,12 +1,59 @@
 # TabPanel
 
-TabPanel is a React component to render an ARIA-compliant TabPanel. It has two sections: a list of tabs, and the view to show when tabs are chosen. When the list of tabs gets focused, the active tab gets focus (the first tab if there isn't one already). Use the arrow keys to navigate between tabs AND select the newly focused tab at the same time.
+TabPanel is a React component to render an ARIA-compliant TabPanel. 
 
-TabPanel is a Function-as-Children component. The function takes the active tab object as as an argument.
+TabPanels organize content across different screens, data sets, and interactions. It has two sections: a list of tabs, and the view to show when tabs are chosen. 
+
+![The “Document” tab selected in the sidebar TabPanel.](https://wordpress.org/gutenberg/files/2019/01/s_E36D9C9B8FFA15A1A8CE224E422535A12B016F88884089575F9998E52016A49F_1541785098230_TabPanel.png)
+
+
+# Table of contents
```

Comment:
Heading hierarchy needs updating — everything except the title needs to get pushed down a level (so this would be an h2, etc.)

Reply:
Should be fixed in 10f7c21